doc/howto_release.md

```diff
@@ -86,10 +86,10 @@ git status
 git show
 ```
 
-Push the tag to the upstream repo:
+Push the tag to the upstream repo, release branch:
 
 ```bash
-git push upstream
+git push upstream releasebranch_8_2
```

Comment:
This is pushing the commits on the release branch. I'm not sure about the tag, probably not. I don't know why this says tag actually. There is no tag created above, so I guess I copy-pasted some other text.

Reply:
This is where I recently messed it up (I tried to use this procedure for the G78 RC3 release but failed).